covid-19-support/src/components/ResourceMap.vue

```diff
@@ -99,6 +117,26 @@ export default {
     boundsUpdated(bounds) {
       this.$emit('bounds', bounds)
     },
+    resetError() {
+      this.showError = false
+      this.errorMessage = ''
+    },
+    getUserLocation() {
+      var map = this.$refs.covidMap.mapObject
+      map.locate({ setView: true, enableHighAccuracy: true })
+      map.on('locationfound', (locationEvent) => {
+        if (locationEvent.latitude && locationEvent.longitude) {
+          this.userLocationData = latLng(locationEvent.latitude, locationEvent.longitude)
+          this.centerUpdated(this.userLocationData)
+        }
```

Comment:
Could you also add a new map marker at this point to indicate where the user is, I don't know if this is built in functionality with LeafletJS or if we just add a normal marker with our own icon?

Reply:
I've added a marker for this scenario